Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "KeyboardKit",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13)
+    ],
+    products: [
+        .library(
+            name: "KeyboardKit",
+            targets: ["KeyboardKit"]),
+    ],
+    targets: [
+        .target(
+            name: "KeyboardKitObjC",
+            path: "KeyboardKit/ObjCSources",
+            exclude: ["ResponderChainDebugging.m"],
+            publicHeadersPath: ""),
```

Comment:
Is there a reason for this empty string?

```suggestion
        ),
```

Reply:
Without explicitly setting this path (i.e. it being the default value of nil), SwiftPM uses the default umbrella directory component of `include` and appends this value to the target path (`KeyboardKit/ObjSources`), giving `KeyboardKit/ObjCSources/include` as the public headers path.

Setting it to an empty string ensures `KeyboardKit/ObjCSources` is used as the public headers directory.